spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/src/main/java/org/springframework/cloud/data/module/deployer/lattice/ReceptorTaskModuleDeployer.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.data.module.deployer.lattice;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.receptor.actions.RunAction;
+import org.cloudfoundry.receptor.client.ReceptorClient;
+import org.cloudfoundry.receptor.commands.TaskCreateRequest;
+import org.cloudfoundry.receptor.commands.TaskResponse;
+import org.cloudfoundry.receptor.support.EnvironmentVariable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.cloud.data.core.ModuleDeploymentId;
+import org.springframework.cloud.data.core.ModuleDeploymentRequest;
+import org.springframework.cloud.data.module.ModuleStatus;
+import org.springframework.cloud.data.module.deployer.ModuleDeployer;
+
+/**
+ * @author Patrick Peralta
+ * @author Mark Fisher
+ * @author Michael Minella
+ */
+public class ReceptorTaskModuleDeployer implements ModuleDeployer {
+
+	private static final Logger logger = LoggerFactory.getLogger(ReceptorTaskModuleDeployer.class);
+
+	public static final String DOCKER_PATH = "docker:///springcloud/stream-module-launcher";
+
+	private final ReceptorClient receptorClient = new ReceptorClient();
+
+	@Override
+	public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
+		ModuleDeploymentId id = ModuleDeploymentId.fromModuleDefinition(request.getDefinition());
+		String guid = guid(id);
+
+		TaskCreateRequest task = new TaskCreateRequest();
+		task.setLogGuid(guid);
+		task.setTaskGuid(guid);
+		task.setRootfs(DOCKER_PATH);
+
+		RunAction runAction = task.getAction().get("run");
+		runAction.setPath("java");
+		runAction.addArg("-Djava.security.egd=file:/dev/./urandom");
+		runAction.addArg("-jar");
+		runAction.addArg("/module-launcher.jar");
+
+		List<EnvironmentVariable> environmentVariables = new ArrayList<EnvironmentVariable>();
+		Collections.addAll(environmentVariables, task.getEnv());
+		environmentVariables.add(new EnvironmentVariable("MODULES", request.getCoordinates().toString()));
+		environmentVariables.add(new EnvironmentVariable("SPRING_PROFILES_ACTIVE", "cloud"));
```

Comment:
I *think* we can remove this here AND in the LRP deployer now that the spring-cloud-stream-modules projects support profile activation automatically. cc @ilayaperumalg